.github/scripts/classify-issue.py

```
@@ -0,0 +1,202 @@
+    for ty, kws in rules["keyword_type"].items():
+        if any(kw_hit(k, tl) for k in kws):
+            return ty
+    return None
+
+
+def enforce_tiers(rules, labels):
+    """Drop violations of each tier's `max`, keeping the highest-precedence member."""
+    tier_of, tier_max = rules["tier_of"], rules["tier_max"]
+    prec = rules["precedence"]
+    by_tier, out = {}, set()
+    for l in labels:
```

Ruff (0.16.2)

[error] Ambiguous variable name: `l`

(E741)